docs/api/en/materials/MeshBasicMaterial.html

```diff
@@ -78,8 +78,10 @@ <h3>[property:Texture aoMap]</h3>
 
 		<h3>[property:Float aoMapIntensity]</h3>
 		<p>
-			Intensity of the ambient occlusion effect. Default is `1`. Zero is no
-			occlusion effect.
+			Intensity of the ambient occlusion effect. Typical ranges are 0-1, where
```

Comment:
> Range is [0, 1].

//

(the only valid values.)

Reply:
Changed, removed 'typically' phrasing. 👍🏻